src/test/resources/amazonReviewSnippets_GroundTruth_vader.tsv

```diff
@@ -188,7 +188,7 @@
 23_3	0.0	0.792	0.208	0.4389	i had hoped this was bought from walmart so that i could return it!
 23_4	0.0	1.0	0.0	0.0	it sends discoutinous signals to tv, so the tv has the pictures for one moment and has only the white noise for the other moment.
 23_5	0.117	0.883	0.0	-0.2755	the other thing i don't like about this 2600 is that it makes the color look over saturated.
-23_6	0.0	0.598	0.402	0.9151	maybe it's good for tvs with not so good color settings, but not good for my tv that already has very vibrant pictures.
+23_6	0.0	0.598	0.402	0.4341	maybe it's good for tvs with not so good color settings, but not good for my tv that already has very vibrant pictures.
```

Comment:
Could you point out which commit fixed the bug with text having duplicate words?

Reply:
595f9cb6788b6ba80e94259e4504facbbfca50cb included this fix, but I only realized that afterwards when the unit test failed. The indexOf() call that was used previously always returned the first occurrence of a word, not the index of any subsequent duplicates.